ext_emconf.php

```diff
@@ -22,8 +22,8 @@
     'version' => '1.0.0',
     'constraints' => array(
         'depends' => array(
-            'typo3' => '7.6.0-7.6.99',
-            'php' => '5.3.0',
+            'typo3' => '10.4.0-10.4.99',
```

Comment:
If we do it like this we would never have released a version that is TYPO3 9 compatible. 
Would these changes work with TYPO3 9 as well? If so I would adjust the version constrains.

Reply:
I changed the `typo3` constraint back to `9.5.0-10.4.99`, so it can be installed with TYPO3V9. The tests can't be executed with TYPO3 V9, because testing-framework has changed a lot from V4.x to 6.x and to get tests running in both versions takes too much time for an example extension which should not be used in production.